test/js/css-url-image-value.js

```diff
@@ -46,4 +55,23 @@ suite('CSSURLImageValue', function() {
       done();
     };
   });
+
+  test('Can set and get correct CSSURLImageValue from StyleMap', function(done) {
+    var inlineStyleMap = this.element.styleMap();
+    inlineStyleMap.set("background-image", new CSSURLImageValue('resources/1x1-green.png'));
+    var urlImageValue = inlineStyleMap.get("background-image");
+    assert.instanceOf(urlImageValue, CSSURLImageValue);
+    assert.equal(urlImageValue.url, "resources/1x1-green.png");
+
+    var oldOnload = urlImageValue._image.onload;
+
+    urlImageValue._image.onload = function() {
+      oldOnload();
```

Comment:
This is getting kind of repetitive. Can you create a function for chaining functions together? Something like this perhaps (maybe you can think of something more general?):

```javascript
function chain(firstFn, secondFn, opt_this) {
  return function() {
    firstFn.call(opt_this);
    secondFn.call(opt_this);
  };
}
```